content/actions/automating-builds-and-tests/building-and-testing-ruby.md

```diff
@@ -54,9 +54,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Set up Ruby
-        uses: ruby/setup-ruby@477b21f02be01bcb8030d50f37cfec92bfa615b6
+        uses: ruby/setup-ruby@359bebbc29cbe6c87da6bc9ea3bc930432750108
```

Comment:
Could you add a comment like
https://github.com/actions/starter-workflows/blob/2e489c26196ef3b0c1ff38e460beae83e522e2d4/ci/ruby.yml#L27-L29

We've seen many people copying and then being surprised they don't get the latest Ruby versions.
That comment would help I believe, and it's from the official starter workflow.

Reply:
Done